src/fsharp/service/service.fsi

```diff
@@ -304,6 +304,10 @@ type public FSharpParsingOptions =
       LightSyntax: bool option
       CompilingFsLib: bool
       IsExe: bool
+
+      /// When true, error messages will include suggestions for parsing errors based on what the compiler knows.
+      /// This is an expensive operation that should be false outside of code fixes, command-line builds, and in-IDE builds.
+      SuggestNamesForErrors: bool
```

Comment:
Do we really need it in parsing options? Suggesting names doesn't happen at parsing stage, does it?

Reply:
Nope. I'll revisit this on Monday